test/ui-test/lightspeedAuthUiTest.ts

```diff
@@ -18,146 +18,145 @@ import {
 } from "./uiTestHelper";
 import { expect } from "chai";
 
-export function lightspeedUILoginTest(): void {
-  describe("Login to Lightspeed", () => {
-    let workbench: Workbench;
-    let explorerView: WebviewView;
-    let modalDialog: ModalDialog;
-    let dialogMessage: string;
-    let sideBar: SideBarView;
-    let viewControl: ViewControl;
-    let adtView: ViewSection;
-
-    before(async () => {
-      // Enable Lightspeed and open Ansible Light view on sidebar
-      workbench = new Workbench();
-      const settingsEditor = await workbench.openSettings();
-      await updateSettings(settingsEditor, "ansible.lightspeed.enabled", true);
-      await updateSettings(
-        settingsEditor,
-        "ansible.lightspeed.URL",
-        process.env.TEST_LIGHTSPEED_URL,
-      );
+before(function () {
```

Comment:
Are these disabled?

Reply:
Hum, they should be enabled only on MacOS. Let's me change that.